includes/admin/class-amp-editor-blocks.php

```diff
@@ -135,9 +135,13 @@ public function enqueue_block_editor_assets() {
 			wp_enqueue_script(
 				'amp-editor-blocks-build',
 				amp_get_asset_url( 'js/amp-blocks-compiled.js' ),
-				array( 'wp-blocks', 'lodash', 'wp-i18n', 'wp-element', 'wp-components' ),
+				array( 'wp-editor', 'wp-blocks', 'lodash', 'wp-i18n', 'wp-element', 'wp-components' ),
 				AMP__VERSION
 			);
+
+			if ( function_exists( 'wp_set_script_translations' ) ) {
+				wp_set_script_translations( 'amp-editor-blocks-build', 'amp' );
```

Comment:
Should this also support the previous versions, too?

Reply:
It looks like this could use the `wp.i18n.setLocaleData()` that's [lower in the function](https://github.com/Automattic/amp-wp/pull/1634/files/ad105928431545d1eb9a5fe00e0abf57a3daa70c#diff-af86f8db0fb2fe24a8aea92de5ee5128R166), if I understand this correctly.

Reply:
You're right!